.github/workflows/FormatCheck.yml

```
@@ -0,0 +1,44 @@
+name: format-check
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags: '*'
+  pull_request:
+
+jobs:
+  check-format:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1]
+        julia-arch: [x86]
+        os: [ubuntu-latest]
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+
+      - uses: actions/checkout@v4
+      - name: Install JuliaFormatter and format
+        # This will use the latest version by default but you can set the version like so:
+        #
+        # julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter", version = "0.13.0"))'
+        #
+        # TODO: Change the call below to
+        #       format(".")
+        run: |
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter"))'
+          julia  -e 'using JuliaFormatter; format(["examples", "src", "test"])'
```

```suggestion
        run: |
          julia  -e 'using Pkg; Pkg.add(PackageSpec(name = "JuliaFormatter"))'
          julia  -e 'using JuliaFormatter; format(".")'
```
Since there is no "baggage" yet, I recommend to just format everything from the get-go